docs/tutorials/setting-up-python-pycharm.md

```diff
@@ -23,10 +23,11 @@ this up, head over to our [SSH tutorial](/docs/tutorials/connecting-to-ev3dev-wi
 ## Configure code synchronization
 
 {% include icon.html type="info" %}
-The first half of this tutorial assumes you are using the free version of PyCharm.
-However, if you have access to the Professional Version of PyCharm, which is free for students
-and some open-source projects, you should probably read the PyCharm Professional version
-alternatives shown later.
+The first half of this tutorial assumes you are using the free "Community"
+version of PyCharm. However, if you have access to the "Professional" version of
+PyCharm, which is free for students and some key open-source projects, you we
```

Comment:
I still think we should just not mention "some key open-source projects" because it is confusing. In fact, this statement is wrong. It is free to a *limited number* key *contributors* of *verified* open source projects (for use on said open source project, not for general use). In other words, there is a 99% chance that readers of this tutorial will not find this information useful.